src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorIT.java

```diff
@@ -441,31 +448,36 @@ public void testMultipleMerge() {
+        assertEquals("[IgmEntity(key=IgmEntityKey(processUuid=51111111-f60e-4766-bc5c-8f312c1984e4, date=2019-05-01T09:00, tso=ES), status=VALIDATION_SUCCEED, networkUuid=7928181c-7977-4592-ba19-88027e4254e5, caseUuid=7928181c-7977-4592-ba19-88027e4254e5, replacingDate=null, replacingBusinessProcess=null, boundaries=[f1582c44-d9e2-4ea0-afdc-dba189ab4358, 3e3f7738-aab9-4284-a965-71d5cd151f71]), " +
+                        "IgmEntity(key=IgmEntityKey(processUuid=51111111-f60e-4766-bc5c-8f312c1984e4, date=2019-05-01T09:00, tso=FR), status=VALIDATION_SUCCEED, networkUuid=7928181c-7977-4592-ba19-88027e4254e4, caseUuid=7928181c-7977-4592-ba19-88027e4254e4, replacingDate=null, replacingBusinessProcess=null, boundaries=[f1582c44-d9e2-4ea0-afdc-dba189ab4358, 3e3f7738-aab9-4284-a965-71d5cd151f71]), " +
+                        "IgmEntity(key=IgmEntityKey(processUuid=51111111-f60e-4766-bc5c-8f312c1984e4, date=2019-05-01T09:00, tso=PT), status=VALIDATION_SUCCEED, networkUuid=7928181c-7977-4592-ba19-88027e4254e6, caseUuid=7928181c-7977-4592-ba19-88027e4254e6, replacingDate=null, replacingBusinessProcess=null, boundaries=[f1582c44-d9e2-4ea0-afdc-dba189ab4358, 3e3f7738-aab9-4284-a965-71d5cd151f71]), " +
+                        "IgmEntity(key=IgmEntityKey(processUuid=21111111-f60e-4766-bc5c-8f312c1984e4, date=2019-05-01T09:00, tso=ES), status=VALIDATION_SUCCEED, networkUuid=7928181c-7977-4592-ba19-88027e4254e5, caseUuid=7928181c-7977-4592-ba19-88027e4254e5, replacingDate=null, replacingBusinessProcess=null, boundaries=[f1582c44-d9e2-4ea0-afdc-dba189ab4358, 3e3f7738-aab9-4284-a965-71d5cd151f71]), " +
+                        "IgmEntity(key=IgmEntityKey(processUuid=21111111-f60e-4766-bc5c-8f312c1984e4, date=2019-05-01T09:00, tso=FR), status=VALIDATION_SUCCEED, networkUuid=7928181c-7977-4592-ba19-88027e4254e4, caseUuid=7928181c-7977-4592-ba19-88027e4254e4, replacingDate=null, replacingBusinessProcess=null, boundaries=[f1582c44-d9e2-4ea0-afdc-dba189ab4358, 3e3f7738-aab9-4284-a965-71d5cd151f71])]",
```

Comment:
Maybe use the static UUID fields defined at the beggining in all these assertEquals assertions ?